### ISSUE TITLE
Scheduling website directly eventually causes raw HTML to show

### ISSUE THREAD

**Issue by kaden-rise** [Rise-Vision/rise-launcher-electron#675]
**To reproduce:**

1. In an empty Schedule, add a URL playlist item that links to this site: 
http://ysm-flyerboard.azurewebsites.net/now/?site=8

2. Assign that Schedule to a Display and let the presentation run for a few hours. For me it happened within 24 hours.

**What happens:**
Eventually the website stops cycling through its stages and displays raw HTML code instead of the site

**What should happen:**
The site should continue to cycle without displaying raw HTML code

**Comment by kaden-rise** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Possible Bug - Scheduling webpages showing raw html](https://trello.com/c/q7aix6Ha/7405-possible-bug-scheduling-webpages-showing-raw-html)

**Comment by stulees** [comment]
@kaden-rise What are the machine specs that you identified this issue on? Were you able to recreate on all platforms?

**Comment by kaden-rise** [comment]
@stulees The device that I was able to reproduce the issue on was a Core i3 7100U NUC with 4GB of RAM. The device is running Windows 10 64-bit. As of right now I haven't reproduced the issue on Ubuntu, but I working on it.

**Comment by stulees** [comment]
@kaden-rise The raw HTML is displayed within the iframe that the page is loaded into. There is nothing malformed or broken in the body structure of page (Viewer) and logs do not indicate an issue with Player.

We set the sandbox attribute on the iframe to enable extra restrictions on the content that is loaded into it. We relax those restrictions by allowing: "allow-forms allow-same-origin allow-scripts".

It could be that the website being loaded is trying to do something that is outside of what we allow, or there is functionality in their source that is breaking the start up of their page. There is nothing that indicates an issue with our Player. Closing, thanks.
